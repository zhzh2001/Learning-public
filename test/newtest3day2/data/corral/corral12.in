436 500
224761233438708192 925389341850499875
344891813648020770 246525252677292576
82959308390111521 390261998230770399
615684636643444551 954253490213532062
765682087569241771 716984030617582275
707482708687456595 515475210095624029
346893547169536239 70301988365486369
803772603694787289 291345550401102097
676044150945655823 434157265807221381
724462979852817186 329375416134923540
165419202472259477 595677251613373183
59620391349741011 680766896408606259
321948958267322350 892645368194506911
908205516030657137 371744093602710723
73690240518659248 633631405298852687
3156120511879998 270839182745751997
485307645305290467 463113790042343577
780007291208272274 700692461397860381
93479032861833281 788171488924819230
463484421061759195 389698284628946010
925277649773369267 637795031040359392
187309260268579139 788594385805377481
938105628487616989 688475593363674125
865400980452908257 735675469674647345
729199766826180091 677866559902184526
617255340032818342 528120193864774165
408469041683293064 6028324600622378
583375883237617558 463200019263970222
580493046001191865 368319672425438290
994442174841702757 453019717920666013
724566008201825891 700867222150133792
678872435045173162 359376602555309151
746475414417709581 962120821016114687
859197323818195370 1722840514780299
944278266227366757 950169126630050257
495826055398901738 197732163177186707
814719359325110446 493763506114795339
880236205775271795 470029569037821977
478304475195267576 767180775061877682
221576430704050773 56799911907701846
129330168590438185 803928714022289110
562722794897718822 67362669378352240
170904652378320385 964238191897010315
376813633020519528 355213060573243187
954390421787400546 317410635954446679
285219553757235634 638039685784568360
714355321527662934 837231266184996762
376661407840560173 533301620798151195
47423853223529206 238050097139852301
164899138252681774 303727894780768349
392569560921341030 721753374222784874
523071583556844022 652721320004186096
623558021747066942 380369779803363467
940896609589576521 60605356711796742
406869308147608135 682986327036015736
494963183312186921 73205406788160465
905671644050553468 284831374071355728
993181233017476759 905101601245212686
937256780730045613 814866666325406372
532191637025000019 325659484349789292
124400002655036415 184925059880851126
143941041156383548 991441168135466672
378677191463436385 898097127495462956
235003392167769110 134285664225386657
532709175576743242 626093455088153321
783600472528937957 950191956433931511
501475171159200608 56737148202316806
61772524526188418 591399556316692303
902032629160454960 836363488804534083
877640701915419796 601948128382589318
399112350748402543 375149924489725568
289899186757073613 739337522692980292
739480998238086440 61377420730289638
942360148204559627 312152423134217897
719932221657481117 998474010547142972
820927118022281091 625714203294295212
751898824385540001 31229822890804511
131635147625806084 739643800726278941
730081192692455003 677309748933129003
640928366849424654 322943669786668446
640897184730788701 686521787396258419
891112818282724278 476750407710278803
513893171268324486 242494261008565896
667668114985575652 845818826828000650
79053154727779779 454209571644598033
5440368023531854 325834936654162847
550602956751144756 864975463406884218
518226351793807624 672468075870482485
728908617027960645 934472178228070723
699749002036578254 204403422731848774
255152603327314528 293608525174405407
61351373706811657 637714768064930041
739314415838945681 17122367902006079
691303535982350040 429735255224122548
538014214193005838 606604770502791293
527894174511551406 234499278743265117
8356028203518678 851615015491880147
557488724111949866 495469594750792014
128671171162038284 697752811668461561
899260023580838662 975051130084289051
362167012375161961 528364103365157823
906646624863346617 566040562447614040
822558714490511550 932238540719044025
655107354507696716 364031440627009655
491870831618305061 515927306789341256
906399358544185698 404626169316631375
897883435806718668 556864568443333838
466963337388764814 776034052715119544
81020806767218607 216801720054651633
161205102032002959 220518534534632286
528675982539464854 7853615739033116
112136682520470824 778319779976134911
734953216722842963 355771220283140338
177185079866340997 583339413538679262
423268106572070884 183439312140803090
719930623786090539 121142861872228158
253470411890630459 145287915822606097
946613929120465791 593210951986515128
403163068827749460 271784948712131769
154862638901105319 86251284489963086
39971361218269060 217466919973447774
836122302558195794 609528151580767280
457893883184802054 627201117222807638
971977946405608607 561686173614325611
448148722627238639 564232650309205920
888787605984738236 57432097739611344
220315761634624672 441305397654599310
646040486106565147 908458879604054526
104839471839398177 983020548553665724
93084605584297534 540783944033432395
581488522093453841 215962062205614844
374376281928932664 797683390310661702
349362748307559982 539184485400320635
298363109944926363 208875742642878836
990209479816076545 920714659245572817
340685593033638352 852723147967931417
853995768858215155 124518891596840982
687440681631199348 509772632217906020
760823340476640025 329751779303531528
704857289838239343 638093989791169118
751514976563527963 238491653036033875
18656814212437843 883133935002046766
899193220095524492 3433138596926419
516612018351422530 911851558118823122
471110638639157164 7010667649535592
822616336657467074 1412116790554008
738029370638682351 825540493046887403
600598639993738088 289871707233023638
196378525476844846 722385042464795525
683193298644223583 415062587554997374
971226944652710279 582883597068865675
446555505142655988 222877708045232526
167233216160209256 787379520238151340
218395029889140591 831663165086934472
700760166907352880 799327995256749229
651405623161294311 643921882459226499
776232921138928560 516322833677411034
267753233276540163 896337997686879305
954473386540204507 881798443022493217
456662246409701718 302411746925714993
975225079014855064 487546804491383079
868535132379468815 692088050083088100
762291805861800397 388289518214139072
979899224700454810 160234493888742465
905898537508157066 859781189070970432
589115497800592687 767971936222095092
594866645387752728 786940320343101221
15561079927851046 174120660856299141
667145516507263466 907482307362563093
444661241365530322 792179511190435851
995094924678736442 773680467459563848
855048617987076788 112482443443111688
240198660877670990 931045484809059304
207258213457567773 44268767892074422
391168047210803031 238295623841006732
38039487325861468 978584556594764966
188905160201397560 924862946279729049
3631888898756639 64551570988366257
607540483194060039 360092354489555281
263330404428510658 987146800424330802
728087738464540601 692396089211208807
833571970783343780 217142380244913935
702951525252031444 589937686446576677
469291032401002465 163501902714286477
637253004678154543 71111995389250883
429136132875664483 386189930096905740
159415642732396895 275724694186390524
965490068377565900 682957417159408394
285989532381671971 964851707974816298
966554094016371807 262711884263634150
382855646912763739 628316188842518351
57038737857362136 723011627155841454
448640194826365040 391463406455461548
834199060178517801 98535239502550517
561550287958831553 92927427861316090
500237782905913873 167795701291137168
230930054756570854 27387203949852150
890259855576441681 765985726107839250
942604521603165236 295053078326998772
408052957233467185 768639217239690873
890248232349930650 918757616729228459
437359357339748383 883996712590985463
805146985804760760 54213340235514730
93984374757426834 400179035210662294
152190110762574164 10279225481937070
142947970462252255 320038623424984740
981410529793927741 898352185586041669
876107168367694823 424773509849841505
43361206947479057 64023178078188260
951411929176469863 53706323152071777
134669224258765146 910716079275017033
725198413096761849 82064135244846525
856610494424848925 277039654376212190
277022290409608786 998096816227688745
805839083227357858 211312374262954234
694022275681466771 797503073518687780
514310323113774800 119413067180244390
108227582245906876 56881613826492026
501888802962393910 220790855651008093
464730335591760571 925342730821897706
607375163628836520 977581814912125400
734291624900904952 64806618340657862
636922457897517269 107148920190137748
759301030687409490 554084377269265826
878197304559591155 96639246845512388
139394156056578451 663749685203103501
281217053168588384 763719366595816377
23471145202441648 876909879733462715
92603590423217774 917634094001633467
976772353398202035 642093702695613911
454803820914410916 146189948647250171
843600652591429952 819029954881473383
354236602158570173 74244579511412374
414358963551293825 68553042840197733
104918403255158386 464973680253405594
237193771171665711 571318802353301800
405214851808263556 366617969345517359
460700804343451646 510478826990558838
470571206670538696 539988167516018757
915855207714858818 996449195943472298
926380503260066468 681535869920284685
14928872619029435 835401918085001700
994520966658668599 109871188522266682
76011554783187256 273908459800653654
548508252451036798 824227545230306560
86242710964167301 837398028989031112
680318785802803370 715739829933707981
856261542054732093 610193974325959933
440064994063087114 450639467110637554
890094271846883830 710536389313751909
640292828494091355 556203294477310512
76902421836367248 516140474444673799
5927162582836856 576612114654425513
571849750998885039 677900907696223957
462650302071395243 458759993968025711
654574904636224020 223499138312029913
530381508484254810 473840784649338654
771749139076949134 720234910922963936
577607964442535051 826069334821550241
455214080865138118 430571848338127731
370444706217003100 227493329539505335
644373403575531815 250571810706861753
271994235515709744 153443600148643604
618505569245225475 932191693446049827
537996863112655897 613915481479788719
827521903103603424 841270775539489809
770957273478812555 949299113517966836
827810866566911730 478674648759587975
315789949684294245 404770372924052451
608952685891586706 517247438584535678
315362839061846921 318173520802499013
953509133372484625 148879323162389342
427680450462256258 555298688735895627
814148150331257003 252084462393644395
158543874968425064 232705519794749051
477288415769321822 188695141039861572
810864618896307798 653254794317362460
846925505326857440 815818334107440184
981906690371307461 54944035704496470
427288810869019510 675014095110122763
753669029763068532 432861793636289089
815876220438681769 134628536522777401
502324493553377584 98175475727485240
60461387201139236 65864969817205213
755676233417504936 50019509568284222
622253129865900329 819326536154239541
30670408937938395 665990311252209619
74934619459750403 848951597798134193
942061071331336668 846371215730476020
438863792015313774 284435284491693065
912172143974641843 692067000703167745
721184970094933895 915089069136367450
746933442994458526 678187583883863084
891648120268942685 360645035475921118
626086741450743612 598312566956817462
9641741331341579 857477565219763975
740389405367265154 413305263942591925
879252105926503816 25143953439588889
198017378944601163 982069638901614144
876136320931965739 974964095484130010
419282229001265563 908168973310758292
482752159106576079 875205092589612691
329096636954813707 886335837539021528
630613558658990506 334614138788586218
764800850698760453 295870207046606882
313721466369334585 91541789777600689
22254471291360576 839172954960108725
822955897237243616 374306445924124005
858903880301035201 720935285491630183
475534335148633860 489746031923600363
930743901597044164 484519724989355169
3374805549021536 873363988107245880
135194110012698240 927645476273611218
913935489672839567 955956084579247672
225861148903349128 37850448818542204
368037158435688070 394587563094327603
439889402159870921 744556415847443151
954502461787920059 803726478041270913
697288507580162743 19967081728507918
939833436578722049 314907282460917664
970270285670149031 908602988713747753
871643460640496874 491172443650517522
503121724047985703 611860017381311132
654862328215316916 975472648156483184
698921628898630148 282182648634119704
719598803433200646 459847964054887471
387539809362728746 918319692554542127
959859463443099442 130838675261035384
203806725110467150 208904549431003138
316971105317469765 232502927299393674
115528482635456850 461135994525950313
491496324042584876 740455669785816636
269979993924580859 634678921746393730
358502738658752298 184384959984817900
433024359880137832 373839154417493757
278280545490197283 13757385826733729
649694639969933425 803829325985482568
508609809153587405 769535324624285148
275667535917319055 694789393600450477
965302449484516895 809056137623711041
104896146881157126 637780076766950506
280695109731582306 329076721548255063
368944008499140992 339298004859307298
267572744670138216 730190483666567033
349352400800833488 342585844533680341
770382797868797595 612707834254196920
520164447505605484 125742852574970402
759601778453176660 833844517405631425
681847321706729362 239128316773973694
48278748838868803 83495516695765175
596602030482168145 132361425323802486
425053502467303788 675207798371396129
612685023779406236 585037429057654783
504418012336365020 596505569728792864
151223548842562615 85095160029172257
653627847916274331 366121252519574944
607961107832782763 891331640771079994
616079871447816945 704804624509128388
340526413676137631 50279638323622465
678714789128809391 441964412264606134
268324676367108905 648089124888394794
6026632231359138 49098272218127405
467382119320611844 345530210561033428
736985344165972371 700942934198475731
236275106738521059 593704245415749923
802073988973811014 641338893112018106
712313284704342661 568476173921146146
835679286241548194 988134467563761144
417661607161366758 11836209512166538
570965668456398575 781502084962803611
705014486890070003 258983551792964340
250167293643855383 866338419876605819
770451416140900344 805695281235388750
82683250316322056 851064971852345666
618772613519827401 171700076676895116
623148644491572311 53803969423050289
943847431533517884 56487087755730157
466597839316685625 964790892212514993
908127806642714944 171829349635785774
777056931014924402 779981650095599721
528018147004175883 837582505830975856
115439001831502339 601120418059491144
211358579380364807 668203449186896738
749214544159942286 888839575466781755
514190124089358017 609509692299176388
935587941399591954 190326413208519634
323636841047582273 774783112621425156
515189339286662110 354401975415808484
899015300110781250 189424047494078952
667836082636352074 473657237751755896
35475217129128296 272241119548654077
163913590869846046 744298198524298066
888711522388225466 485595187006748326
594038318681384773 942429647024073405
929627287945779207 686991726988441569
457353390263782425 341149405304110812
821399278312299603 35473990805715466
25384667196998307 565756534345286461
635644757751815352 619447641207677918
880255223452667674 198304662924658338
455041667523337513 331409089905250277
289715463749661099 595814599836479016
649149029135169664 658120974519609445
446971907448975803 537599371100136815
510165345649459498 617894825818126193
771782056518672552 742966580953758521
370413794705490314 131352090273505278
545541838373746010 80663482753015152
358482475328901290 236767051426624601
863785502298970885 935762515303893314
793784304652593202 613875987852255937
940592898030163305 986850655129796022
330907220802977405 764077313436319803
714976156888102653 578464567460826727
232345584205448833 150040984333403944
15042714520325419 594108221264220333
538954076195958298 60706772155532946
175948473367239170 90936331782692582
385826693528471512 626843112395177315
363013862793229624 891032920412387298
921646695114279803 874845290156511087
730980377697562014 40634746350373163
714164642195418492 105319230609735700
428720757158773337 644482146892367320
818545804112512636 382578375401477011
595170221643676767 814069059987129149
78601234143457870 208625299909689435
474771538355230914 715266640906728882
775452139700298969 459609528982392041
835664712107065331 456719110447061177
887191673836865407 419099968561995804
924425017809447902 506313173842412097
571849409518140156 290824803220034629
762864086546293839 214612496500546147
277154462732036673 103068961491822257
427582192103554509 112870949575902825
789825554386805865 882766740265688773
964227573783112383 541898607594371324
148322134202470693 127711232676965513
530717791709244830 606772405263626294
912482536956603443 421787214729061107
699981500930543693 674245944158728469
87384149611188765 876021093778937551
329650250032928635 904788493600009179
436271829674630899 761304263287572124
286541315639406022 463962936327173506
213397953084732964 451788469875102078
33316814884064665 57192861583599082
267959756020166627 927973207449765837
393258784752425117 672825028336756463
206345291911950870 997678706518706476
712523159971537739 926091002877479121
995348148653146381 381756751594521282
912440092420954016 709485245440310438
660319226906639580 124225722982447065
13591110545365360 188728687750625526
157666415861167003 321256712654934340
460428163223484423 281377834692688330
772567803385805361 479556340937729136
63236937245504093 739383405659122720
28472573200000796 579507857280563130
972380957947178976 525333112943804584
760494769558546080 174842303867815661
156367288344343710 654575886124441359
631961453471949430 500178674546419835
951214117409467195 283006196759763409
807609520553973229 306288015665962619
752077455251425685 811456323169292172
174291757672022902 853091504597593424
193954962469912853 897878994411241028
109047590999961645 100208210963032696
505553963669003438 522586088341912399
468147473997000220 229061814378575681
472756626689058811 359308231266030352
685989322491096884 409458566563336210
430516189147606885 965274259079710645
84165645532849749 47131507169322879
43202326116947265 429014630211424319
249305972179691355 152300621617455001
689956876083945304 656077644794608087
38935573233640172 956224569944838887
476762189864867853 556662469913529819
714587272764255307 854012149188972761
788449039633839503 256489877878505706
992893626885765271 59628917626262569
848202806004687577 553266370149778604
646077114733484921 244232281302618648
186685068089362159 815115412393882532
41585443382476744 214217674050202455
103920783781616412 295121915662127981
247369341183133730 74518900114727270
916239186545151386 498556134831249411
629361235461675081 41142190821630199
605373103660313278 52965162054143392
929796295071996407 836823007869030190
847515893074006324 514642332462216933
709826438254353509 766701495223644033
494557669727081289 678743875114860322
31194427030520840 305447017667032477
318766960398426040 167573768966828141
