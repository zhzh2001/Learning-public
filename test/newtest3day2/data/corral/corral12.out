927423810261168442
